python/strings_udf/strings_udf/__init__.py

```diff
@@ -87,6 +88,29 @@ def _get_ptx_file():
         return regular_result[1]
 
 
+# Maximum size of a string column is 2gb
+STRINGS_UDF_DEFAULT_HEAP_SIZE = 2**31
+STRINGS_UDF_HEAP_SIZE = os.environ.get(
+    "STRINGS_UDF_HEAP_SIZE", STRINGS_UDF_DEFAULT_HEAP_SIZE
+)
+heap_size = 0
```

Comment:
I would prefix these with underscores unless you want them to be visible. I could see arguments for either of the first two being visible, but they will also be settable so it's safer to hide them and provide getters if needed later.

Reply:
Very similar to @bdice's query above.